external-crates/move/move-compiler/src/to_bytecode/translate.rs

```diff
@@ -663,8 +664,23 @@ fn function(
     )
 }
 
-fn visibility(v: Visibility) -> IR::FunctionVisibility {
+fn visibility(
+    context: &mut Context,
+    function_name: &FunctionName,
+    entry: Option<Loc>,
+    v: Visibility,
+) -> IR::FunctionVisibility {
     match v {
+        _ if is_public_for_testing(
```

Comment:
nit: I usually think of `is_` functions in rust returning a `bool`, so naming this something like `public_testing_visibility` might be clearer